web/settings/external_ties.go

```diff
@@ -52,3 +53,22 @@ func (h *HTTPHandler) getExternalTies(c echo.Context) error {
 
 	return jsonapi.Data(c, http.StatusOK, doc, nil)
 }
+
+func (h *HTTPHandler) redirectToPremium(c echo.Context) error {
+	if !middlewares.IsLoggedIn(c) {
+		return middlewares.ErrForbidden
+	}
+	inst := middlewares.GetInstance(c)
+
+	// TODO: move to a SettingsService method to avoid the dependency on the
+	// instance module.
+	url, err := inst.ManagerURL(instance.ManagerPremiumURL)
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest, err.Error())
+	}
+	if url == "" {
+		return echo.NewHTTPError(http.StatusNotFound, "This instance does not have a premium subscription manager.")
```

Comment:
I think we should show an html page. You can grep `error.html` in the code to have some examples.